src/coreclr/jit/lowerxarch.cpp

```diff
@@ -1614,7 +1566,7 @@ void Lowering::LowerHWIntrinsicCreate(GenTreeHWIntrinsic* node)
 
             tmp1 = comp->gtNewSimdHWIntrinsicNode(TYP_SIMD16, op1, NI_Vector128_Create, simdBaseJitType, 16);
             BlockRange().InsertAfter(op1, tmp1);
-            LowerNode(tmp1);
+            tmp1 = LowerNode(tmp1)->gtPrev;
```

Comment:
I don't think this pattern of assuming that `LowerNode(x)->gtPrev` will return the lowered `x` is acceptable, it is not in the contract of `LowerNode`.

I think we can we avoid it by attaching the node to its user before lowering. It will also avoid the need to call `ClearUnusedValue` below.

Reply:
Fixed.